### ISSUE TITLE
Russian SMS

### ISSUE THREAD

**Issue by alabotski** [jd1378/otphelper#98]
Can't parse such text in SMS
```
Сеансовый пароль Интернет-банка e-bgpb.by: 0000. Никому не сообщайте!
```

**Comment by jd1378** [comment]
is `Сеансовый пароль` the sensitive keyword in this message? 
does it repeat in normal messages that does not contain a code ?

**Comment by alabotski** [comment]
Yes `пароль` is sensitive keyword

For example https://github.com/tianma8023/XposedSmsCode
I can set my word for RegEx

It will be good to have such option COPYSMSCODE

![Screenshot_20240608-181809](https://github.com/jd1378/otphelper/assets/439337/525137f3-20fd-49ed-829b-eabe490f48d9)

**Comment by jd1378** [comment]
fixed in 1.16.0
